zzk/snapshot/snapshot_test.go

```diff
@@ -104,9 +104,9 @@ func (t *ZZKTest) TestSnapshotListener_Listen(c *C) {
 	c.Log("Sending failure snapshot")
 	serviceID = "service-id-failure"
 	if nodeID, err := Send(conn, serviceID); err != nil {
-		c.Errorf("Could not send failure snapshot: ", err)
+		c.Errorf("Could not send failure snapshot: %s", err)
 	} else if err := Recv(conn, nodeID, &actual); err != nil {
-		c.Errorf("Could not receive failure snapshot: ", err)
+		c.Errorf("Could not receive failure snapshot: %s", err)
```

Comment:
Hat's off for plowing through this. I'm super excited to get go vet going. (Get it? GOing? I kill me.) Let's discuss the break label pattern you've been using on Tuesday. My main concerns are that it seems to make things a little more complex to read and debug, with no clear benefit (apparent to me). If you teach me something and convince me otherwise, I will mergify all the things. :)